hack/release.sh

```diff
@@ -1,16 +1,44 @@
-#!/bin/bash -e
+#!/bin/bash
 
-git tag $TAG
-git push https://github.com/nmstate/kubernetes-nmstate $TAG
+set -xe
 
-$GITHUB_RELEASE release -u nmstate -r kubernetes-nmstate \
-    --tag $TAG \
-	--name $TAG \
-    --description "$(cat $DESCRIPTION)"
+tag=$(hack/version.sh)
 
-for resource in "$@" ;do
+function upload() {
+    resource=$1
     $GITHUB_RELEASE upload -u nmstate -r kubernetes-nmstate \
         --name $(basename $resource) \
```

Comment:
The arguments should be all indented on the same level, what is this cascade! :D